conda/recipes/cudf_kafka/meta.yaml

```
@@ -43,17 +44,22 @@ requirements:
   host:
     - python
     - cython >=0.29,<0.30
+    - cuda-version ={{ cuda_version }}
     - cudf ={{ version }}
     - libcudf_kafka ={{ version }}
     - setuptools
   run:
     - python
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
     - libcudf_kafka ={{ version }}
     - cudf ={{ version }}
 
 test:
   requires:
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
```

Is `cuda-version` sufficient for constraining `cudatoolkit` ([ref]( https://github.com/conda-forge/cuda-version-feedstock/blob/3ce1bae701079226f982b173667edc5d0a64c01f/recipe/meta.yaml#L17 ))?

```suggestion
    - cudatoolkit
```